src/image_gen/imagegenutils.hpp

```diff
@@ -27,6 +27,7 @@
 #pragma warning(pop)
 
 #include "imagegenpipelineargs.hpp"
+#include "src/http_payload.hpp"
```

Comment:
Fwd declare MultiPartParser & rapidjson::Dcoument instead of HttpPayload?

Reply:
partly done - removed HttpPayload fwd (not needed)  - added MultiPartParser fwd.
however, for rapidjson i just simply included it in the header file